script.js

```diff
@@ -1,12 +1,44 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let img;
+let backgroundImg;
+function preload() {
+  img = loadImage('./images/turtle.svg');
+  backgroundImg = loadImage('./images/beach.svg')
+}
 
 function setup() {
-  createCanvas(windowWidth, windowHeight);
-    background(255);
+  createCanvas(displayWidth, displayHeight);
+  backgroundImg.resize(windowHeight,windowWidth - 200)
+  background(backgroundImg)
+  noStroke()
+ 
+  
 }
 
 function draw() {
-  noStroke();
-  fill(random(colorlist));
-  ellipse(mouseX, mouseY, 25, 25);
+  img.resize(200,200);
+  cursor(image(img, mouseX-20,mouseY-20))
```

Comment:
I think I know what you were trying to do here, but to use image as cursor in p5 you have to put the string path to the image, just like we did with CSS.